src/Policy/CallbackPolicyIncludingPath.php

```diff
@@ -5,7 +5,6 @@
 namespace Chubbyphp\Deserialization\Policy;
 
 use Chubbyphp\Deserialization\Denormalizer\DenormalizerContextInterface;
-use Chubbyphp\Deserialization\DeserializerLogicException;
 
 final class CallbackPolicyIncludingPath implements PolicyInterface
```

Comment:
Shall we rename "back" to `CallbackPolicy `in v3?

Reply:
done